src/test/java/org/assertj/core/api/Assertions_linesOf_Test.java

```diff
@@ -29,6 +30,7 @@
   @Test
   public void should_read_lines_of_file_with_UTF8_charset() {
     File file = new File("src/test/resources/utf8.txt");
+    assumeTrue("src/test/resources/utf8.txt file is blank. Skipping test", file.length() > 0);
```

Comment:
I'm not sure to understand @eas5, we don't need to assume the file is blank, we know it is not and it it was the following assertion would fail anyway. 

Can you clarify?

Cheers

Reply:
Sure!

The assumption is `file.length() > 0`, so we always expect the file to exist and have content. The error message _"src/test/resources/utf8.txt file is blank. Skipping test"_ will be passed to the AssumptionViolatedException in case of failure, which will halt and skip the test instead of failing it.

Reply:
In my eyes, this assumption would hide a potential problem. The test should fail if the file does not exist, we should not skip it.

(BTW `org.assertj.core.api.Assumptions.assumeThat` should be favored)

Reply:
I agree wih @scordio and thus closing this PR.